snap/snapcraft.yaml

```diff
@@ -24,16 +24,17 @@ compression: lzo
 
 apps:
   scummvm:
-    command-chain: ["snap/command-chain/alsa-launch"]
+    command-chain:
+      - snap/command-chain/alsa-launch
+      - snap/command-chain/desktop-launch
```

Comment:
This (`desktop-launch`) is added by the extension, so by including it here we're running it twice

Reply:
On my systems nothing observable. But I shouldn't be dropping it in _this PR_.

Will restore.